preliz/tests/test_beta_mode.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+import numpy as np
+from preliz import beta_mode
+
+
+@pytest.mark.parametrize(
+    "lower, upper, mode, mass, expected_alpha, expected_beta",
+    [
+        (0.25, 0.75, 0.5, 0.9, 6.41, 9.12),  # Example test case
```

Comment:
the value of alpha and beta seems to be right. this should be a symmetrical distribution and hence alpha and beta should be equal.